instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -296,4 +296,47 @@ private static boolean supportsMultipleMarkers() {
 
     return true;
   }
+
+  /**
+   * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+   * any time.
+   */
+  public static final class Builder {
+    private boolean captureExperimentalAttributes;
+    private List<String> captureMdcAttributes = emptyList();
+    private boolean captureCodeAttributes;
+    private boolean captureMarkerAttribute;
+    private boolean captureKeyValuePairAttributes;
+    private boolean captureLoggerContext;
+
+    Builder() {}
+
+    public void setCaptureExperimentalAttributes(boolean captureExperimentalAttributes) {
```

Comment:
Nit: if you returned `Builder` you could chain these invocations in e.g. `LogbackSingletons`